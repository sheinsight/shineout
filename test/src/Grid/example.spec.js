import exampleTest from '../../example'

describe('Grid[snapshot]', () => {
  exampleTest('Grid')
})
