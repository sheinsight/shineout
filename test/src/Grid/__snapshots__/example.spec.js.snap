// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid[snapshot] Snapshot test: Grid[site/pages/components/Grid/example-01-base.tsx] 1`] = `
<div>
  <div>
    <div />
    <div>
      <div>
        <div />
        <div />
      </div>
    </div>
    <div>
      <div>
        <span>
          1
        </span>
      </div>
      <div>
        <span>
          2
        </span>
      </div>
      <div>
        <span>
          3
        </span>
      </div>
      <div>
        <span>
          5
        </span>
      </div>
      <div>
        <span>
          8
        </span>
      </div>
      <div>
        <span>
          13
        </span>
      </div>
      <div>
        <span>
          21
        </span>
      </div>
      <div>
        <span>
          34
        </span>
      </div>
      <div>
        <span>
          55
        </span>
      </div>
    </div>
  </div>
  <div />
  <div>
    <div>
      1/5
    </div>
  </div>
  <div>
    <div>
      2/5
    </div>
  </div>
  <div>
    <div>
      3/5
    </div>
  </div>
  <div>
    <div>
      4/5
    </div>
  </div>
  <div>
    <div>
      5/5
    </div>
  </div>
</div>
`;

exports[`Grid[snapshot] Snapshot test: Grid[site/pages/components/Grid/example-02-offset.tsx] 1`] = `
<div>
  <div>
    With 1/3, Offset 1/3
  </div>
</div>
`;

exports[`Grid[snapshot] Snapshot test: Grid[site/pages/components/Grid/example-03-nested.tsx] 1`] = `
<div>
  <div>
    1/2
  </div>
  <div>
    <div>
      1/2
    </div>
    <div>
      <div>
        1/3
      </div>
      <div>
        1/3
      </div>
      <div>
        1/3
      </div>
    </div>
  </div>
</div>
`;

exports[`Grid[snapshot] Snapshot test: Grid[site/pages/components/Grid/example-04-gutter.tsx] 1`] = `
<div>
  <div>
    <div>
      <div>
        1/8
      </div>
    </div>
    <div>
      <div>
        1/8
      </div>
    </div>
    <div>
      <div>
        1/8
      </div>
    </div>
    <div>
      <div>
        1/8
      </div>
    </div>
    <div>
      <div>
        1/8
      </div>
    </div>
    <div>
      <div>
        1/8
      </div>
    </div>
    <div>
      <div>
        1/8
      </div>
    </div>
    <div>
      <div>
        1/8
      </div>
    </div>
  </div>
</div>
`;
